docs/landingpage.html

```diff
@@ -0,0 +1,178 @@
+		<div class="container-fluid">
+			<div class="row">
+				<div class="col-md-9">
+					<a class="au-header__brand" href="#">
+						<img class="au-header__brand-image" alt="Insert alternate text here" src="./assets/img/header-logo-agov.png">
+						<div class="au-header__text">
+						<h1 class="au-header__heading">Example Site</h1>
+						<div class="au-header__subline">
+							An example of the system in use
+						</div>
+						</div>
+					</a>
+				</div>
+			</div>
+		</div>
+	</header>
+
+	<!-- MAIN NAVIGATION -->
+	<nav class="au-main-nav au-main-nav--dark">
+		<div class="container-fluid">
+			<div class="row">
+				<div class="col-md-12">
+					<div class="au-main-nav__content" id="main-nav-default-alt">
+						<button
+							aria-controls="main-nav-default-alt"
+							class="au-main-nav__toggle au-main-nav__toggle--open"
+							onClick="return AU.mainNav.Toggle( this )">
+							Menu
+						</button>
+						<div class="au-main-nav__menu">
+							<div class="au-main-nav__menu-inner">
+								<div class="au-main-nav__focus-trap-top"></div>
+								<button
+									aria-controls="main-nav-default-alt"
+									class="au-main-nav__toggle au-main-nav__toggle--close"
+									onClick="return AU.mainNav.Toggle( this )">
+									Close
+								</button>
+								<ul class="au-link-list">
+									<li><a href="/index.html">Home</a></li>
+									<li><a href="/about.html">About</a></li>
+									<li class="active"><a href="/landingpage.html">Landing page</a></li>
+									<li><a href="/newpage.html">New Page</a></li>
+								</ul>
+								<div class="au-main-nav__focus-trap-bottom"></div>
+							</div>
+						</div>
+						<div class="au-main-nav__overlay" aria-controls="main-nav-default-alt" onClick="return AU.mainNav.Toggle( this )"></div>
+					</div>
+				</div>
+			</div>
+		</div>
+	</nav>
+
+	<!-- BREADCRUMBS -->
+	<div class="container-fluid au-body main-content">
+		<div class="row">
+			<div class="col-md-12">
+				<nav class="au-breadcrumbs" aria-label="breadcrumb">
+					<ul class="au-link-list au-link-list--inline">
+						<li>
+							<a href="#">Home</a>
+						</li>
+						<li>Landing page</li>
+					</ul>
+				</nav>
+			</div>
+		</div>
+		<!-- content section -->
+		<div class="container">
+			<div class="row card__container">
+				<section class="col-xs-12 col-sm-6 col-md-4 col-lg-4 card">
+					<h1 class="au-display-lg">Landing page</h1>
+					<p>A landing page lets you show all the sections inside of a top level navigation item</p>
+				</section>
+				<section class="col-xs-12 col-sm-6 col-md-4 col-lg-4 card card__linked">
+					<a href="/landingpage/subpage.html">
+						<h2 class="au-display-lg">Sub page</h2>
+						<p>This is a card that links to the subpage.</p>
+						<i class="card__carat"></i>
+					</a>
+				</section>
+				<section class="col-xs-12 col-sm-6 col-md-4 col-lg-4 card card__linked">
+					<a href="/landingpage/subpage.html">
```

Comment:
Maybe make a `subpage2.html`?  It would also make it easier for our users, so they don't have to create another page